src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -907,6 +907,65 @@ public override IOperation VisitWhileLoop(IWhileLoopOperation operation, int? ca
             return null;
         }
 
+        public override IOperation VisitVariableDeclarationGroup(IVariableDeclarationGroupOperation operation, int? captureIdForResult)
+        {
+            // We erase variable declarations from the control flow graph, as variable lifetime information is contained in a parallel data structure.
+            foreach (var declaration in operation.Declarations)
+            {
+                VisitVariableDeclaration(declaration, captureIdForResult);
+            }
+
+            return null;
```

Comment:
>return null; [](start = 12, length = 12)

Is IVariableDeclarationGroupOperation always used as a statement? Consider asserting that it is equal to the ```_currentStatement```.

Reply:
It's not used as the statement for `for` loops.

---
In reply to: [166783785](https://github.com/dotnet/roslyn/pull/24692#discussion_r166783785) [](ancestors = 166783785)

Reply:
> It's not used as the statement for for loops.

It well be. We are not handling loops properly at the moment. Please add an assert.

---
In reply to: [166784997](https://github.com/dotnet/roslyn/pull/24692#discussion_r166784997) [](ancestors = 166784997,166783785)

Reply:
To clarify, we cannot return null, unless we are visiting this node in a statement mode.

---
In reply to: [167016477](https://github.com/dotnet/roslyn/pull/24692#discussion_r167016477) [](ancestors = 167016477,166784997,166783785)

Reply:
Added.

---
In reply to: [167018923](https://github.com/dotnet/roslyn/pull/24692#discussion_r167018923) [](ancestors = 167018923,167016477,166784997,166783785)